tools/webpack/blocks.js

```diff
@@ -65,15 +65,66 @@ module.exports = function( env = { environment: 'production', watch: false, buil
 	const config = {
 		...baseConfig( env ),
 		entry: {
-			'file/view': normalizeJoin(baseDir, `node_modules/@wordpress/block-library/build-module/file/view` ),
-			'navigation/view': normalizeJoin(baseDir, `node_modules/@wordpress/block-library/build-module/navigation/view` ),
-			'navigation/view-modal': normalizeJoin(baseDir, `node_modules/@wordpress/block-library/build-module/navigation/view-modal` ),
-			'search/view': normalizeJoin(baseDir, `node_modules/@wordpress/block-library/build-module/search/view` ),
+			'navigation/view': normalizeJoin( baseDir, 'node_modules/@wordpress/block-library/build-module/navigation/view' ),
+			'image/view': normalizeJoin( baseDir, 'node_modules/@wordpress/block-library/build-module/image/view' ),
+			'query/view': normalizeJoin( baseDir, 'node_modules/@wordpress/block-library/build-module/query/view' ),
+			'file/view': normalizeJoin( baseDir, 'node_modules/@wordpress/block-library/build-module/file/view' ),
+			'search/view': normalizeJoin( baseDir, 'node_modules/@wordpress/block-library/build-module/file/view' ),
 		},
 		output: {
 			devtoolNamespace: 'wp',
-			filename: `[name]${ suffix }.js`,
-			path: normalizeJoin(baseDir, `${ buildTarget }/blocks` ),
+			filename: `./blocks/[name]${ suffix }.js`,
+			path: normalizeJoin( baseDir, buildTarget ),
+			chunkLoadingGlobal: `__WordPressPrivateInteractivityAPI_${ Math.floor( Math.random() * 1000 ) }__`,
```

Comment:
@mikachan, this is random, so it will cause an update to the version number in the asset file. We need to have it stable to move forward:

```suggestion
			chunkLoadingGlobal: `__WordPressPrivateInteractivityAPI__`,
```

Let's update this line, and run `npm run build:dev` again. It should become stable after that. I haven't tested, but it's the only reason I can think of.

Reply:
Yes, that worked locally! Thank you! Committing now...

Reply:
@luisherranz, the part that makes the global’s name dynamic needs to be more predictable. It can’t change on every individual build.

Reply:
Oh, ok. I was wondering why that was removed 😄

Well, I introduced it to discourage people from trying to hack their way into accessing those modules, but Webpack also auto-generates semi-random IDs for the import of each individual file and those can also change randomly between builds, so it's not that necessary. We can leave it out 🙂

Reply:
>  I was wondering why that was removed 

That was the simplest way I could figure out in a short time that wouldn't impact Beta 1.

> We can leave it out

Thanks for confirming, there is no further action necessary.